src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -184,103 +184,111 @@ def __init__(
         del argvals["self"]
         if argvals["start_timeout"] is None:
             argvals["start_timeout"] = 60
-        for arg_name, arg_values in argvals.items():
-            setattr(self, arg_name, arg_values)
-        self.argvals = argvals
-        self.new_session = self.mode.value[0]
+        self.new_session = argvals["mode"].value[0]
         self.file_transfer_service = file_transfer_service
 
-    def __call__(self):
-        if self.lightweight_mode is None:
+        if argvals["lightweight_mode"] is None:
             # note argvals is no longer locals() here due to _get_session_info() pass
```

Comment:
```suggestion
```
This code comment is no longer relevant and can be removed

Reply:
Done. Thank you.